front-end/src/Profile.css

```diff
@@ -78,3 +77,40 @@
   background-color: #2D2D2D;
   color: #3CDD1B;
 }
+
+#buttons {
+    display: flex;
+    justify-content: space-around;
+}
+
+.filter-button {
+    margin: 2em;
+    font-size: 18px;
+    border: 2px solid #84ED6F;
+    background-color: #666666;
+    color: #84ED6F;
+    padding: 1em 2em 1em 2em;
+}
+
+fieldset{
```

Comment:
I explain why I'm requesting this a bit in my general PR comment, but could you possibly change this to refer to the settings fieldset specifically, in some way? Perhaps assign the settings fieldset an id and refer to it here, because this styling somehow interferes with the Individual Show page's.